src/main/api/exchangeOAuthCodeForAccessTokenUsingPKCE.json

```diff
@@ -0,0 +1,64 @@
+{
+  "uri": "/oauth2/token",
+  "comments": [
+    "Exchanges an OAuth authorization code for an access token.",
+    "If using the Authorization Code grant, you will make a request to the Token endpoint to exchange the authorization code returned from the Authorize endpoint and a code_verifier for an access token."
```

Comment:
This was already here, but perhaps should be re-written, not sure what it means. We are using the authorization code grant, otherwise we wouldn't be using this method.

Reply:
Or if you want to leave that is fine - I can re-word some of this stuff later, I don't want to hold you up too long fixing my technical debt. :-)

Reply:
OK. Was trying to keep this in line with the other `exchange` method, defined in `exchangeOAuthCodeForAccessToken.json` but will rewrite both to make it clearer.

Reply:
No worries, wasn't hard to change the verbiage a bit.